onnxruntime/core/providers/cuda/reduction/reduction_ops.cc

```diff
@@ -887,5 +887,6 @@ REGISTER_KERNEL_INT32(ReduceMean)
 REGISTER_KERNEL_INT32(ReduceProd)
 REGISTER_KERNEL_INT32(ReduceSum)
 
+REGISTER_KERNEL_TYPED_12(ReduceProd, int64_t)
```

Comment:
Wouldn't this try to register for opset-12 as well ? (Not sure if this is causing the build failures as I don't think ORT has taken in the ONNX commits pertaining to opset-12 yet)

Reply:
Oh I see. Fixed it. Thanks a lot.